javascript/ql/lib/semmle/javascript/dataflow/internal/TaintTrackingPrivate.qll

```diff
@@ -18,14 +19,22 @@ predicate defaultAdditionalTaintStep(DataFlow::Node node1, DataFlow::Node node2)
     ContentSet::arrayElement(), node2.(FlowSummaryNode).getSummaryNode())
 }
 
+private class SanitizerGuardAdapter extends DataFlow::Node instanceof TaintTracking::AdditionalSanitizerGuardNode
```

Comment:
That class no longer has subclasses in our own codebase. It was used internally to compute interprocedural barriers, but this has been refactored into `DerivedBarrierGuardNode` to avoid computing that code path when using dataflow2.

But perhaps there are external uses of the class. I've added a layer of backwards compatibility in `DataFlowPrivate`. Note that the shared data flow library has no concept of default barrier nodes; all we can do is try to remove the affected nodes from the data flow graph.